hosting/src/pages/Home.tsx

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 
-import { Toolbar } from '@material-ui/core';
+import { Toolbar, Link } from '@material-ui/core';
+import Rule from '../components/Rule';
+import CreateRoom from '../components/CreateRoom';
+import BugForm from '../components/BugForm';
 
 const Home: React.FC = () => (
-  <>
+  <div>
     <Toolbar />
-    <div>contents</div>
-  </>
+    <Rule />
+    <CreateRoom />
+    <BugForm />
+    <Link href="https://github.com/kanko-dori/code-jinro">GitHub</Link>
```

Comment:
```suggestion
    <Container>
      <Rule />
      <CreateRoom />
      <BugForm />
    </Container>
```

Containerで囲って左寄りを回避してください